src/annotator/integrations/pdf.js

```diff
@@ -29,6 +30,21 @@ import { PDFMetadata } from './pdf-metadata';
 // is enough room. Otherwise, allow sidebar to overlap PDF
 const MIN_PDF_WIDTH = 680;
 
+/**
+ * Return true if `anchor` is in an un-rendered page.
+ *
+ * @param {Anchor} anchor
+ */
+function anchorIsPlaceholder(anchor) {
```

Comment:
I worry again about the conflation of _placeholder_ to mean both the single placeholder-per-page in which anchors are quarantined until after render _and_ a single anchor-highlight that is within said one-per-page placeholder...

Reply:
I renamed this in a later commit to `anchorIsInPlaceholder`.